README.md

```diff
@@ -171,6 +175,59 @@ The `profile` object provides a set of attributes corresponding to user attribut
 
 Update your Callback URL in our Yoti Dashboard and you should be good to go!
 
+## AML Integration
+
+Yoti provides an AML (Anti Money Laundering) check service to allow a deeper KYC process to prevent fraud. This is a chargeable service, so please contact [sdksupport@yoti.com](mailto:sdksupport@yoti.com) for more information.
+
+Yoti will provide a boolean result on the following checks:
+* PEP list - Verify against Politically Exposed Persons list
+* Fraud list - Verify against  US Social Security Administration Fraud (SSN Fraud) list
+* Watch list - Verify against watch lists from the Office of Foreign Assets Control
+
+To use this functionality you must ensure:
+* Your application is assigned to your Organisation in the Yoti Dashboard - please see [here](https://www.yoti.com/developers/documentation) for further information.
+* Within your application please ensure that you have selected the 'Given Name(s)' and 'Family Name' attributes from the Data tab. This is the minimum requirement for the AML check.
```

Comment:
This part is slightly different to Seb's same text for [Ruby](https://github.com/getyoti/yoti-ruby-sdk/pull/14/files#diff-04c6e90faac2675aa89e2176d2eec7d8R201)

Reply:
Yes Seb changed the doc after my commit.